SumDU/App Delegate/AppDelegate.swift

```diff
@@ -15,11 +17,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
     
+    NetworkActivityIndicatorManager.shared.isEnabled = true
+    NetworkActivityIndicatorManager.shared.startDelay = 0.0
+    
+    // Initialize CoreData stack
+    let _ = CoreDuck.quack
+    CoreDuck.printErrors = true
+    
     self.window = UIWindow(frame: UIScreen.main.bounds)
     
     if UIDevice.current.userInterfaceIdiom == .pad, let window = self.window {
+      // Search
       let searchViewController = SearchViewController()
+      searchViewController.fetchData()
+      
+      // Schedule
       let scheduleViewController = ScheduleViewController()
+      if let firstHistoryObject = searchViewController.history.fetchedObjects?.first {
+        scheduleViewController.fetchSchedule(for: firstHistoryObject)
+        searchViewController.selectedObjectID = firstHistoryObject.id
+        
+        // Send request
+        let networkingManager = NetworkingManager()
+        networkingManager.delegate = scheduleViewController
+        networkingManager.scheduleRequest(for: firstHistoryObject)
+      }
+      
       let splitViewController = UISplitViewController()
```

Comment:
Subclass UISplitViewController and move all setup logic for containing view controllers there.